ci-operator/config/openstack-k8s-operators/openstack-baremetal-operator/openstack-k8s-operators-openstack-baremetal-operator-main.yaml

```diff
@@ -0,0 +1,18 @@
+build_root:
+  from_repository: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make gotest
+  container:
+    from: src
+zz_generated_metadata:
+  branch: master
```

Comment:
thanks

i will also add the pre commit job since we have the config already at https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/157607c01176d01c57a1d4420474a10bc13f791a/.pre-commit-config.yaml#L1

Reply:
done